test/ChessBoardTest.hs

```diff
@@ -21,4 +23,17 @@ test_filter =
                                 , (('C', 6), Queen Black)
                                 , (('D', 8), King Black) ] Ongoing) (filterChessBoard board2 Black)
 
+
```

Comment:
These tests did not pass for me - it could be because of the invalid character error mentioned above